apisix/plugins/prometheus.lua

```diff
@@ -29,7 +29,6 @@ local _M = {
     version = 0.2,
     priority = 500,
     name = plugin_name,
-    init = exporter.init,
```

Comment:
why do we need to delete this line? I think we need to keep it.

Reply:
@membphis 
Because we don't call the prometheus init function when the plugin reload anymore.